app/uk/gov/hmrc/cdsreimbursementclaim/models/claim/CompleteClaim.scala

```diff
@@ -55,7 +55,7 @@ object CompleteClaim {
     maybeContactDetailsAnswer: Option[CompleteContactDetailsAnswer],
     maybeBasisOfClaimAnswer: Option[CompleteBasisOfClaimAnswer],
     maybeCompleteBankAccountDetailAnswer: Option[CompleteBankAccountDetailAnswer],
-    supportingEvidenceAnswers: CompleteSupportingEvidenceAnswer,
+    supportingEvidenceAnswer: NonEmptyList[SupportingEvidence],
```

Comment:
it's a good idea to keep this consistent with the FE - over there you used your type alias

Reply:
Done